data/gui/window/campaign_dialog.cfg

```diff
@@ -227,6 +227,41 @@
 											tooltip = _"Sort in approximate chronological order of story events"
 										[/toggle_button]
 									[/column]
+									
+									[column]
+										grow_factor = 1
+										horizontal_grow = true
+
+										[multimenu_button]
+											id = "filter_completion"
+											tooltip = _"Filter by campaign completion status"
+											[option]
+												checkbox = yes
+												label = _ "Not Completed"
+												tooltip = _ "Show incomplete campaigns"
```

Comment:
`#po: campaigns which the player hasn't beaten, rather than campaigns which the author hasn't finished writing`

Reply:
Where should I add it? (No idea about po hints, sorry)

Reply:
Added and also updated tooltip text, please check.

Reply:
You've put it in the right place, on the line directly before the translatable string.

However, I don't think it needs "by player" in the string itself; the player is the target audience by default.